README.md

```diff
@@ -1,6 +1,6 @@
 # Core Motion [![Build Status](https://travis-ci.org/appcelerator-modules/ti.coremotion.svg?branch=master)](https://travis-ci.org/appcelerator-modules/ti.coremotion)
 
-**This module is available only for iOS 7 and above.**
+**This module is available only for iOS 7 and later.**
 
 ## COMPILING AND PACKAGING THE MODULE
 
```

Comment:
remove and change the entire description of the installation of the module here. the git submodule etc. was meant for hyper loop which is not applicable now that you have changed this module.